src/schema/v1/show.ts

```diff
@@ -663,8 +663,19 @@ const Show: GraphQLFieldConfig<void, ResolverContext> = {
       type: new GraphQLNonNull(GraphQLString),
       description: "The slug or ID of the Show",
     },
+    partnerId: {
+      type: GraphQLString,
+      description: "The slug or ID of the Partner",
+    },
   },
-  resolve: (_root, { id }, { showLoader }) => {
+  resolve: (_root, { id, partnerId }, { showLoader, partnerShowLoader }) => {
+    if (id && partnerId) {
```

Comment:
I was thinking we need to differentiate by the `accessToken`, but I guess differentiating by args passed in makes sense. I'd suggest making the argument or the description more clear. It's tricky to describe it, but maybe "Partner ID or slug of the show to fetch detailed representation".

I almost feel we should eventually consolidate the two Gravity endpoints: [show](https://github.com/artsy/gravity/blob/2e667561217b8da7e22e5beab55385dbc3ac6b05/app/api/v1/shows_endpoint.rb#L16) and [partner_show](https://github.com/artsy/gravity/blob/2e667561217b8da7e22e5beab55385dbc3ac6b05/app/api/v1/partners_shows_endpoint.rb#L49).

Reply:
Yes if we either consolidate the gravity endpoint params or get the id from the [grandparent](https://github.com/artsy/metaphysics/pull/1856#pullrequestreview-268056777) if it is possible, then the argument is not necessary and and it is gonna be ugly to differentiate the loader based on the presence of `partnerId`.